config/initializers/omniauth.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
   provider :google_oauth2, ENV["GOOGLE_CLIENT_ID"], ENV["GOOGLE_CLIENT_SECRET"], {
-    hd: "gpmail.org" # restrict logins to only Gpmail accounts
+    hd: %w(gpmail.org acts2.network) # restrict logins to only gpmail.org or acts2.network accounts
```

Comment:
from my research in the previous thread, I think this actually doesn't work and is a documentation bug. But yeah, haven't tested 🤷‍♂️

Reply:
Oh OK - actually yeah maybe the whole domain is still gpmail.org, but just some different email addresses.